test/lib/npm.js

```diff
@@ -178,17 +166,16 @@ t.test('npm.load', async t => {
 
     outputs.length = 0
     logs.length = 0
-    await npm.exec('get', ['scope', '\u2010not-a-dash'])
+    await npm.exec('get', ['scope', 'usage'])
```

Comment:
I'm trying to figure out what code path this was testing, and if it still makes sense as a test?

Reply:
It does test the effect of exec-ing multiple commands to keep `npm.command` set. The unicode dash was probably added on to that test originally. I think it makes sense to keep this one.